src/main/java/org/fundacionjala/sfdc/pages/base/FormBase.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.sfdc.pages.base;
+
+import org.fundacionjala.sfdc.CommonActions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by abelb on 8/8/2017.
```

Comment:
@AT-04/at-04 
You should not add your name and date in the class description.
Please remove it in all the classes. :)